packages/docusaurus/src/client/templates/index.html.template.ejs

```diff
@@ -5,8 +5,10 @@
     <meta name="viewport" content="width=device-width">
     <meta name="generator" content="Docusaurus">
     <title><%= htmlWebpackPlugin.options.title %></title>
+    <%= htmlWebpackPlugin.options.headTags %>
   </head>
   <body>
+    <%= htmlWebpackPlugin.options.bodyTags %>
```

Comment:
Since we're already doing this, let's add `postBodyTags` too, so we'll have `preBodyTags` and `postBodyTags`. There are use cases for both.